test/common/compressor/zlib_compressor_impl_test.cc

```diff
@@ -15,63 +19,74 @@ namespace {
 
 class ZlibCompressorImplTest : public testing::Test {
 protected:
-  static const int8_t gzip_window_bits{31};
-  static const int8_t memory_level{8};
-  static std::string multiply32BytesText(uint64_t n) {
-    std::string str{};
-    for (uint64_t i = 0; i < n; ++i) {
-      str.append("Sed ut perspiciatis unde omnis i");
-    }
-    return str;
-  }
+  static const int64_t gzip_window_bits{31};
+  static const int64_t memory_level{8};
+  static const uint64_t default_input_size{796};
 };
 
 class ZlibCompressorImplDeathTest : public ZlibCompressorImplTest {
 protected:
-  static void compressorBadInitTestHelper(int8_t window_bits, int8_t mem_level) {
+  static void compressorBadInitTestHelper(int64_t window_bits, int64_t mem_level) {
     ZlibCompressorImpl compressor;
     compressor.init(ZlibCompressorImpl::CompressionLevel::Standard,
                     ZlibCompressorImpl::CompressionStrategy::Standard, window_bits, mem_level);
   }
 
   static void unitializedCompressorTestHelper() {
-    Buffer::OwnedImpl in;
-    Buffer::OwnedImpl out;
+    Buffer::OwnedImpl input_buffer;
+    Buffer::OwnedImpl output_buffer;
     ZlibCompressorImpl compressor;
-    in.add(multiply32BytesText(1));
-    compressor.compress(in, out);
+    TestUtility::feedBufferWithRandomCharacters(input_buffer, 100);
+    compressor.compress(input_buffer, output_buffer);
   }
 };
 
 TEST_F(ZlibCompressorImplDeathTest, CompressorTestDeath) {
   EXPECT_DEATH(compressorBadInitTestHelper(100, 8), std::string{"assert failure: result >= 0"});
   EXPECT_DEATH(compressorBadInitTestHelper(31, 10), std::string{"assert failure: result >= 0"});
-  EXPECT_DEATH(unitializedCompressorTestHelper(), std::string{"assert failure: result >= 0"});
+  EXPECT_DEATH(unitializedCompressorTestHelper(), std::string{"assert failure: result == Z_OK"});
 }
 
-TEST_F(ZlibCompressorImplTest, CompressGzipEnconding) {
-  Buffer::OwnedImpl in;
-  Buffer::OwnedImpl out;
-  Envoy::Compressor::ZlibCompressorImpl compressor;
+TEST_F(ZlibCompressorImplTest, CallingChecksum) {
```

Comment:
nit: do you mind add/backfilling small // comments on top of each test with what the test is testing? We are trying to do this for new tests.